### DISCUSSION TITLE
Add Components onClick

### DISCUSSION THREAD

**Discussion by tenserebel** [Atri-Labs/atrilabs-engine#724] [Help: Miscellaneous]
Hello, I wanted to ask is there a way we can add components such as text or checkbox through a button? If so, could you direct me to proper resources. 

Thank you.

**Comment by cruxcode** [comment]
Hi @tenserebel, do you mean to add a component dynamically when someone clicks on the button?

**Reply by tenserebel** [reply]
Yes, when a button is clicked a new text should be added.
